### DISCUSSION TITLE
A script is currently running?

### DISCUSSION THREAD

**Discussion by StellariumSentinel** [Stellarium/stellarium#3704] [General]
Dear,
is there a script command to know if a script is currently running? Like a "core.getScriptStatus"...

Thank you so much

**Comment by gzotti** [comment]
How would you call this? From a script? I don't think there is a way to execute more than one at a time.

**Comment by StellariumSentinel** [comment]
You're right, stupid question ... ;'-)
I thought there was a way to solicit a "busy" message from Stellarium...
